zipkin-tracer.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'faraday', '~> 0.8'
   s.add_dependency 'finagle-thrift', '~> 1.4.2'
+  s.add_dependency 'excon', '~> 0.53'
```

Comment:
could this be a development dependency? It will be a tough sell to make everyone to depend on excon